fees/ether-fi/index.ts

```diff
@@ -31,8 +102,33 @@ const getPayoutDetails = async (options, target) => {
   return [asset_eth, rate_eth];
 };
 
+const getSsvRevenue = async (options: FetchOptions) => {
+  const logs = await options.getLogs({
+    target: SSV,
+    eventAbi: "event Transfer(address indexed from, address indexed to, uint256 value)",
+    topic: "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef",
+    fromBlock: await options.getStartBlock() - 30 * 86400 / 12, //claimed every 31 days
```

Comment:
hmm, if it is claimed every 31 days, it is better to leave a spike on the chart every 31st day instead of trying to average it.

So, I would remove from & toBlock here as well

Reply:
Since we are reporting daily revenue and averaging the claimed amount, it actually reflects what was earned each day, even though it wasn’t feasible to claim daily. I think it makes sense to keep it averaged, as it more accurately represents the earnings for that day

Reply:
yes, but it is not really revenue till it is transferred, we apply the same logic for other adapters that we have listed.

Reply:
ok